pom.xml

```diff
@@ -47,7 +76,7 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
-        <version>4.3.8</version>
+        <version>${vertx.version}</version>
         <type>pom</type>
```

Comment:
https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-351
![image](https://github.com/folio-org/edge-orders/assets/113523904/587766be-08eb-4849-8234-fad3d7793379)

Reply:
we should put log4j-bom first

Reply:
good catch, thank you